baldrick/plugins/tests/test_github_towncrier_changelog.py

```diff
@@ -18,6 +18,26 @@
 enabled=true
 """
 
+CUSTOM_TYPE_TEMPLATE = """
+[tool.towncrier]
+    package = "sunpy"
+    directory = "changelog/"
```

Comment:
Is this slash Windows friendly? Is Windows supported in `baldrick`?

Reply:
I honestly have no idea, it's really designed to run server side, only crazy people run windows on a server right....